gst/mqtt/mqttcommon.h

```diff
@@ -21,8 +21,8 @@
 #define GST_MQTT_ELEM_NAME_SINK "mqttsink"
 #define GST_MQTT_ELEM_NAME_SRC "mqttsrc"
 
-#define GST_MQTT_LEN_MSG_HDR    512
-
+#define GST_MQTT_LEN_MSG_HDR          1024
+#define GST_MQTT_MAX_LEN_GST_CPAS_STR 512
```

Comment:
Are you going to make this dynamic later? (non-fixed size of header)

Reply:
> Are you going to make this dynamic later? (non-fixed size of header)

Yes, I am.